src/routes/titles.js

```diff
@@ -26,11 +31,12 @@ module.exports = (fastify, opts, done) => {
         '/api/titles/:epNum',
         {preHandler: adminPreHandler, logLevel: 'warn'},
         async (request, reply) => {
-            const epNum = request.params.epNum;
-            const countedSuggestions = await getSuggestionsWithCountedVotes({
-                epNum,
-            });
-            return reply.send(countedSuggestions);
+            let requestedEpNum = request.params.epNum; // getSuggestionsWCV defaults to current
```

Comment:
or you could have done it here
```js
let requestedEpNum = request.params.epNum || await getCurrentEpNum();
```
Would have made `getSuggestionsWithCountedVotes` simpler